jsonrpc/endpoints_zkevm.openrpc.json

```diff
@@ -411,6 +411,18 @@
           }
         }
       ]
+    },
+    {
+      "name": "zkevm_getLatestGlobalExitRoot",
+      "summary": "Returns the latest global exit root.",
```

Comment:
I would suggest to mention that is the latest GER used in a batch (because it could exists newers GER that are not used yet)